cwms-data-api/src/main/java/cwms/cda/api/Controllers.java

```diff
@@ -105,6 +105,8 @@ public final class Controllers {
     public static final String CATEGORY_ID = "category-id";
     public static final String EXAMPLE_DATE = "2021-06-10T13:00:00-0700[PST8PDT]";
     public static final String VERSION_DATE = "version-date";
+
+    public static final String VERSION_TYPE = "version-type";
```

Comment:
Wondering if "date-version-type" is better as to not confuse with other usages of the name "version" like the freeform strings in a time series id.

Reply:
Done